brainfuck2/bf.cpp

```diff
@@ -29,7 +29,7 @@ class Tape {
 
   inline int get() { return tape[pos]; }
   inline void inc(int x) { tape[pos] += x; }
-  inline void move(int x) { pos += x; while (pos >= tape.size()) tape.push_back(0); }
+  inline void move(int x) { pos += x; while (pos >= tape.size()) tape.resize(2 * tape.size()); }
```

Comment:
looks like little hack, i prefer minimum hacks

Reply:
yes, but still push_back is a public api, its readable and understable call (i'm crystal coder in mind :) ). It should be language care to optimize it as fast as possible, not user. In other implementations, it was users PRs where i also prefer for simplify things.